torch/_inductor/config.py

```diff
@@ -167,6 +167,16 @@
     "TORCHINDUCTOR_MAX_AUTOTUNE_GEMM_BACKENDS", "ATEN,TRITON"
 ).upper()
 
+# Be able to skip gemm so that
+# 1. it's more stable to test triton matmul template in unit test. Otherwise we rely
+#    on the triton template out performing the aten kernel to properly test the
+#    triton template
+# 2. make it easier to repro some max-autotune issue. E.g., we found some max
+#    autotune issue very flaky to repro because the bad triton config get picked
+#    with certain probability. Skipping aten gemm and reducing the config pool
+#    can greatly help reproducing those issues.
+skip_aten_gemm = os.environ.get("TORCHINDUCTOR_SKIP_ATEN_GEMM") == "1"
```

Comment:
You can get this same functionality through `max_autotune_gemm_backends`

Reply:
Didn't realize I can do that earlier:

```
def use_aten_gemm_kernels():
    return (
        not use_max_autotune() or _use_autotune_backend("ATEN")
    ) and not config.skip_aten_gemm

```

But taking another look, yes, I can avoid adding an extra config.